Fix special install of pywbem 1.0.0b1

A recent PR has added support such that the CI runs of pywbemtools with PACKAGE_LEVEL=latest installs the beta version of pywbem 1.0.0b1. However, there is a bug in the Makefile:
```
makefile: Installing pywbem 1.0.0b1 as long as 1.0.0 is not yet released
python -m pip install --upgrade pywbem=1.0.0b1
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
ERROR: Invalid requirement: 'pywbem=1.0.0b1'
Hint: = is not a valid operator. Did you mean == ?
Makefile:418: recipe for target 'install_pywbemtools_py2.7.done' failed
make: [install_pywbemtools_py2.7.done] Error 1 (ignored)
```

--- a/Makefile
+++ b/Makefile
@@ -418,10 +418,11 @@ install_$(package_name)_$(pymn).done: Makefile pip_upgrade_$(pymn).done requirem
 	@echo "makefile: Installing $(package_name) (editable) and its Python runtime prerequisites (with PACKAGE_LEVEL=$(PACKAGE_LEVEL))"
 	-$(call RM_FUNC,$@)
 	-$(call RMDIR_FUNC,build $(package_name).egg-info .eggs)
-# TODO: Remove the following workarouind once pywbem 1.0.0 is releaed.
+# TODO: Remove the following workaround once pywbem 1.0.0 is releaed.
 ifeq ($(PACKAGE_LEVEL),latest)
 	@echo "makefile: Installing pywbem 1.0.0b1 as long as 1.0.0 is not yet released"
-	-$(PIP_INSTALL_CMD) $(pip_level_opts) pywbem=1.0.0b1
+	$(PIP_INSTALL_CMD) $(pip_level_opts) pywbem==1.0.0b1
+	@echo "makefile: Done installing pywbem 1.0.0b1 as long as 1.0.0 is not yet released"
 endif
 	$(PIP_INSTALL_CMD) $(pip_level_opts) -r requirements.txt
 	$(PIP_INSTALL_CMD) $(pip_level_opts) -e .

